sonobuoy/Makefile

```diff
@@ -63,11 +64,31 @@ sonobuoy: bin/sonobuoy
 	./bin/sonobuoy delete
 
 clean:
+	-vagrant destroy -f
```

Comment:
I don't think adding `vagrant destroy` to `clean` is the right thing.
Moreover, it should _not_ ignore the errors from `vagrant` if this were the right place.

Instead, I'd suggest that users should run `vagrant destroy` manually after
running `make run-on-vagrant`.